### PR TITLE
show archived user

### PR DESCRIPTION
**Pull request by mxaly**
Archived users are still in app and it's possible to access their profile page.

### CODE REVIEW

File: app/react/components/user/user-props-stats.jsx [hunk 1]
```diff
@@ -8,7 +8,9 @@ const UserPropsStats = ({userName, propsReceivedCount, propsGivenCount}) => (
     <div className="well well-sm">
       <div className="row">
         <div className="col-sm-6 col-md-8">
-          <h4>{userName}</h4>
+          <h4>
+            {userName} {archived ? '(archived)' : null}
```
Comment (bardzusny) [inline app/react/components/user/user-props-stats.jsx]: How about `{archived && '(archived)'}`?

File: app/repositories/users_repository.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def active
   end
 
   def find_by_id(id)
-    active.find(id)
+    all.find(id)
```
Comment (bardzusny) [inline app/repositories/users_repository.rb]: Doesn't this change make this method redundant? (It basically does nothing now except passing function param).

Comment (mxaly) [inline app/repositories/users_repository.rb]: sort of, but it's nice to have it here in the repository. Let's leave it as it is:)

Comment (bardzusny) [inline app/repositories/users_repository.rb]: :+1:

**Pull request merged**